src/applications/personalization/profile/components/personal-health-care-contacts/Contact.jsx

```diff
@@ -38,18 +57,24 @@ const Contact = ({
     addressLine3,
     addressLine4,
   ].filter(line => !!line);
+
   const isNextOfKin = contactType.match(/next of kin/i);
-  let title = index === 0 ? 'Primary' : 'Secondary';
-  title = `${title} ${isNextOfKin ? 'next of kin' : 'emergency contact'}`;
-  const description = isNextOfKin
-    ? 'The person you want to represent your health care wishes if needed.'
-    : 'The person we’ll contact in an emergency.';
+  const baseContactType = contactType.replace(/^(primary\s+|other\s+)/i, '');
+  let title = '';
+  if (numberOfContacts > 1) {
+    title = index === 0 ? 'Primary ' : 'Secondary ';
+  }
+  title = capitalize(`${title}${baseContactType}`);
```

Comment:
Since you are only rendering the `<h3>` for each contact if there are more than one, do we even need the check for number of contacts here?

suggestion for single liner could be:
`const title = capitalize(`${index === 0 ? 'Primary ' : 'Secondary '}${baseContactType}`);`

Non-blocking, but just wasn't sure if there was a reason for this since the jsx takes care rendering based on length anyway